plugins/mission/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_library(dronecore_mission SHARED
+add_library(dronecore_mission ${PLUGIN_LIBRARY_TYPE}
```

Comment:
The other plugins will need that too, not just the three we have right now.

Reply:
Good point, I'll change that.

Reply:
Actually I'll merge like this and will change that when adding those plugins to `dronecore_server`.

Reply:
Ok, sure!